src/components/PaginatedTable/TableRow.tsx

```diff
@@ -38,7 +38,7 @@ interface LoadingTableRowProps<T> {
 
 export const LoadingTableRow = <T,>({index, columns, height}: LoadingTableRowProps<T>) => {
     return (
-        <tr className={b('row')}>
+        <tr className={b('row', {loading: true})}>
```

Comment:
There are no styles for `row_loading`, do we need this?

Reply:
it's for tests only to handle "table data was loaded"